rosdep/base.yaml

```diff
@@ -2936,7 +2936,10 @@ libzmqpp3:
   ubuntu: [libzmqpp3]
 linux-headers-generic:
   arch: [linux-headers]
-  debian: [linux-headers-generic]
+  debian:
+    jessie: [linux-headers-3.16.0-4-all]
+    sid: [linux-headers-3.16.0-4-all]
```

Comment:
This seems to be the best reference I've found for the default kernel versions: https://packages.debian.org/search?keywords=linux-source